src/sass/paragon.scss

```diff
@@ -1,7 +1,9 @@
 @import "vendor/vendor";
 @import "base/base";
 @import "modules/modules";
-@import "dashboard/dashboard";
+// @import "dashboard/dashboard";
+
+//Don't publish with this on. Thanks.
 @import "experimental/experimental";
```

Comment:
Think it would be smart to always have this line commented out? So by default we are NOT importing it?

Reply:
That's the idea when publishing... but we at least at the moment have to manually turn it on or off while working on it. It has to be on while we work on experimentals though... unless we just import experimentals into the file that using them... which could be possible. Thoughts?

Reply:
Just seems like on `develop` we should make sure it is NOT being imported. So that is why I commented on it here. I could see us manually uncommenting it on a feature/experimental branch. But not bring that into `develop`.